.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,33 @@
+# https://github.com/marketplace/actions/shellcheck
+name: Check shell scripts
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+    paths:
+      - '.github/workflows/**'
```

Comment:
I'm not sure we want the paths check as this will only run on PRs that modify workflows?

Reply:
You're right, this is a copy and waste!